dists/emscripten/build.sh

```diff
@@ -274,12 +369,9 @@ fi
 if [[ "make" =~ $(echo ^\(${TASKS}\)$) || "build" =~ $(echo ^\(${TASKS}\)$) ]]; then
   cd "${ROOT_FOLDER}"
   echo "Running make"
-  emmake make
+  emmake make -j 5
```

Comment:
Please don't hardcode a cpu count.
We use `num_cpus=$(nproc || grep -c ^processor /proc/cpuinfo || echo 1)` in dockerized-bb

Reply:
great, I used that instead